cleanup.md

```
@@ -0,0 +1,18 @@
+- [ ] Refactor `WandererKills.Core.Config` into a single canonical module: remove all duplicate `lib/wanderer_kills/core/config.ex` files and update callers to use the new module.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)